src/ActionList/Item.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import styled from 'styled-components'
+import {AlertIcon} from '@primer/octicons-react'
 import Box, {BoxProps} from '../Box'
+import {Tooltip} from '../drafts'
```

Comment:
When I try and run tests locally and import the "new" Tooltip, I got this mysterious error:

```
 FAIL  src/ActionList/ActionList.test.tsx
  ● Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'LeadingVisual')

      226 | export type NavListLeadingVisualProps = ActionListLeadingVisualProps
      227 |
    > 228 | const LeadingVisual = ActionList.LeadingVisual
          |                                  ^
      229 |
      230 | LeadingVisual.displayName = 'NavList.LeadingVisual'
      231 |

      at Object.LeadingVisual (src/NavList/NavList.tsx:228:34)
      at Object.require (src/NavList/index.ts:1:1)
      at Object.require (src/drafts/index.ts:51:1)
      at Object.require (src/ActionList/Item.tsx:18:1)
      at Object.require (src/ActionList/index.ts:3:1)
      at Object.require (src/index.ts:49:1)
      at Object.require (src/utils/testing.tsx:7:1)
      at Object.require (src/utils/test-matchers.tsx:7:1)
```

Any idea what might be causing this? I couldn't figure it out.